src/UsgsAstroLsStateData.cpp

```diff
@@ -147,7 +149,9 @@ std::string UsgsAstroLsStateData::toJson() const {
         {STATE_KEYWORD[STA_PLATFORM_FLAG], m_PlatformFlag},
         {STATE_KEYWORD[STA_ABERR_FLAG], m_AberrFlag},
         {STATE_KEYWORD[STA_ATMREF_FLAG], m_AtmRefFlag},
-        {STATE_KEYWORD[STA_INT_TIME], m_IntTime},
+        {STATE_KEYWORD[STA_INT_TIME_LINES], m_IntTimeLines},

```

Comment:
This then will, I think, need to be altered to use an vector of arbitrary objects.

Reply:
link below.